webapi/src/test/scala/org/knora/webapi/routing/admin/ProjectsRouteZSpec.scala

```diff
@@ -71,7 +72,7 @@ object ProjectsRouteZSpec extends ZIOSpecDefault {
    */
   private def encode(iri: String) = URLEncoder.encode(iri, "utf-8")
 
-  def spec = suite("ProjectsRouteZ")(
+  def spec: Spec[TestEnvironment with Scope, Any] = suite("ProjectsRouteZ")(
```

Comment:
this kind of type annotation doesn't add much value in my opinion

Reply:
My IDE complains when public member do not have a type annotation. Also https://twitter.github.io/effectivescala/#Types%20and%20Generics-Return%20type%20annotations recommends to add them for public members.